proto/edgehog/deviceruntime/containers/v1/v1.proto

```diff
@@ -110,64 +108,69 @@ message PidsStats {
 // BlkioStats stores all IO service stats for data read and write.
 message BlkioStats {
   // Available on hosts with cgroups v2
-  repeated BlkioValue io_service_bytes_recursize = 1;
+  repeated BlkioValue io_service_bytes_recursive = 1;
   // Available on hosts with cgroups v1
-  repeated BlkioValue io_service_recursize = 2;
+  repeated BlkioValue io_serviced_recursive = 2;
   // Available on hosts with cgroups v1
-  repeated BlkioValue io_queue_recursize = 3;
+  repeated BlkioValue io_queue_recursive = 3;
   // Available on hosts with cgroups v1
-  repeated BlkioValue io_service_time_recursize = 4;
+  repeated BlkioValue io_service_time_recursive = 4;
   // Available on hosts with cgroups v1
-  repeated BlkioValue io_wait_time_recursize = 5;
+  repeated BlkioValue io_wait_time_recursive = 5;
   // Available on hosts with cgroups v1
-  repeated BlkioValue io_merged_recursize = 6;
+  repeated BlkioValue io_merged_recursive = 6;
   // Available on hosts with cgroups v1
-  repeated BlkioValue io_time_recursize = 7;
+  repeated BlkioValue io_time_recursive = 7;
   // Available on hosts with cgroups v1
-  repeated BlkioValue sectors_recursize = 8;
+  repeated BlkioValue sectors_recursive = 8;
 }
 
+// Operation on a block device
 message BlkioValue {
-  uint64 major = 1;
-  uint64 minor = 2;
-  string op = 3;
-  uint64 value = 4;
+  // Device type major
+  optional uint64 major = 1;
+  // Device type minor
+  optional uint64 minor = 2;
+  // Operation on the device
+  optional string op = 3;
+  // Value of the operation
+  optional uint64 value = 4;
 }
 
 // CPU related info of the container
 message CpuStats {
   // All CPU stats aggregated since container inception.
-  CpuUsage cpu_usage = 1;
+  optional CpuUsage cpu_usage = 1;
   // System Usage.
   optional uint64 system_cpu_usage = 2;
```

Comment:
How is this different from the cpu_usage field?

Reply:
This is time spent by the system (e.g. system calls like write, read...)